README.md

```diff
@@ -18,7 +18,7 @@ use chobie\Jira\Issues\Walker;
 
 $api = new Api(
     'https://your-jira-project.net',
-    new Basic('yourname', 'password')
+    new Basic('yourname', 'apitoken')
```

Comment:
Both `password` and `apitoken` would work. It's better to leave existing code and add a comment, e.g.:

```
new Basic('yourname', 'password') // Use `apitoken` instead of password for Jira Cloud instances.
```